config/functions

```diff
@@ -122,6 +122,11 @@ get_full_module_dir() {
   echo "/usr/lib/kernel-overlays/${1:-base}/lib/modules/$(get_module_dir)"
 }
 
+# get full path to firmware dir
+get_full_firmware_dir() {
+  echo "/usr/lib/firmware"
+}
+
```

Comment:
Would not be better to just set variable FIRMWARE_DIR somewhere?

Reply:
That was an intermediate refactoring step. In a later commit this is changed to accept an optional parameter
```
# get full path to firmware dir
# optional parameter specifies overlay level (default is base)
get_full_firmware_dir() {
  echo "$(get_kernel_overlay_dir $1)/lib/firmware"
}
```